src/index.js

```diff
@@ -0,0 +1,129 @@
+import './style.css';
+
+const arrOfObjs = [
+  {
+    name: 'Name',
+    score: '100',
+  },
+  {
+    name: 'Name',
+    score: '20',
+  },
+  {
+    name: 'Name',
+    score: '50',
+  },
+  {
+    name: 'Name',
+    score: '78',
+  },
+  {
+    name: 'Name',
+    score: '125',
+  },
+  {
+    name: 'Name',
+    score: '77',
+  },
+  {
+    name: 'Name',
+    score: '42',
+  },
+];
+
+function htmlContent() {
```

Comment:
* In this project you are encouraged to use ES6 syntax (ie arrow function) [Refer to Project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md#:~:text=using%20webpack%20and%20ES6%20features%2C) i suggest you replace the `function` keyword with `arrow function` as it is a requirement for this project